x-pack/plugins/lens/server/migrations.test.ts

```diff
@@ -174,7 +174,6 @@ describe('Lens migrations', () => {
       index="ff959d40-b880-11e8-a6d9-e546fe2bba5f"
       metricsAtAllLevels=false
       partialRows=false
-      includeFormatHints=true
```

Comment:
We shouldn't make this change here - this is about migrations, and our 7.8.0 saved obejcts did contain this flag.